desc/objectives/linear_objectives.py

```diff
@@ -51,6 +51,12 @@ class FixBoundaryR(_Objective):
     name : str
         Name of the objective function.
 
+
+    Notes
+    -----
+    If specifying particular modes to fix, the rows of the resulint constraint `A`
```

Comment:
Typo: "resulting" instead of "resulint"?

Same typo in `FixBoundaryZ` as well.